src/features/metadata-based-view/MetadataBasedItemList.js

```diff
@@ -127,11 +129,11 @@ class MetadataBasedItemList extends React.Component<Props, State> {
                 );
                 break;
             default: {
-                const data = getProp(item, 'metadata.data', {});
                 const mdFieldName = this.getMetadataColumnName(metadataColumn);
+                const data = getProp(find(fields, ['name', mdFieldName]), 'value'); // find field and get value
```

Comment:
Should we have a default value `{}` just like the origin code?

Reply:
Default value of `getProp()` is `undefined` which is fine in this case.